kaldb/src/test/java/com/slack/kaldb/testlib/KaldbConfigUtil.java

```diff
@@ -5,6 +5,7 @@
 import com.slack.kaldb.proto.config.KaldbConfigs;
 
 public class KaldbConfigUtil {
+  // TODO: Drop this method since we can no longer initialize the chunk manager using this.
```

Comment:
Can we add a `@Deprecated` annotation to this? That will flag it in `IntelliJ`, so folks don't inadvertently add it to new tests before we can remove it.

Reply:
good point. Done.